### DISCUSSION TITLE
Possible values of colorspace returned by get_texttrace()

### DISCUSSION THREAD

**Discussion by little-polka-dot** [pymupdf/PyMuPDF#2514] [Q&A] [answered]
How many possible values can the colorspace parameter have when returned by the get_texttrace function, besides 1 for GRAY and 3 for RGB?

**Comment by little-polka-dot** [comment]
Is the fill and color returned by get_drawings always in RGB format?

**Reply by JorjMcKie** [reply]
> Is the fill and color returned by get_drawings always in RGB format?

Yes, an internal conversion to RGB is being used.

**Reply by little-polka-dot** [reply]
Thanks, And what are the formats of the color returned by get_texttrace()? except 'GRAY' and 'RGB'

**Reply by little-polka-dot** [reply]
There is no detailed explanation in the documentation regarding this.

**Reply by JorjMcKie** [reply, marked as answer]
In the documentation, you will see an example for GRAY. This only happens if the text does not provide a colorspace at all.
RGB (which is the normal case) is given as usual as a tuple of 3 floats in range 0 <= f <= 1. So you will not see any surprise here, do you.
The only thing I am regretting at the design of this function is, that I did not convert color to the sRGB format.
